storage/atlas/versions.tf

```
@@ -0,0 +1,26 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    mongodbatlas = {
+      source  = "mongodb/mongodbatlas"
+      version = ">= 1.10.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.5.1"
```

Avoid approximate constraints within modules

```suggestion
      version = ">= 3.0.0"
```